src/test/kotlin/org/jetbrains/intellij/IntelliJPluginSpec.kt

```diff
@@ -401,19 +422,22 @@ class IntelliJPluginSpec : IntelliJPluginSpecBase() {
         )
     }
 
-    private fun collectTestClassPaths(): Pair<String, String> {
-        buildFile.appendPrintTestClassPathsTasks()
-        return buildAndGetClassPaths("printTestCompileClassPath", "printTestRuntimeClassPath")
+    private fun collectSourceSetClassPaths(sourceSetName: String = "test"): Pair<String, String> {
```

Comment:
If not a fan of the default parameter, happy to pass "test" from original call sites or make an aliased function, just me let know.